camunda-formio-tasklist-vue/docs/components/TaskList.md

```diff
@@ -19,6 +19,8 @@ solution in your Vue.js based project.
 | containerHeight | string | 100vh | Prop to adjust the height values of Vue component, in case of Vue component is too much for your integrated application. You can adjust the height from the range 100-400 |
 | taskSortBy | string | "created" | Prop to decide by default what value should the tasks be sorted when displayed at first. You can use the values like - "created", "dueDate", "follweowUpDate", "priority", "name", "assignee" |
 | taskSortOrder  | string | "desc" | Prop to decide by default what value should the tasks should be ordered when displayed at first. You can use the values like - "desc", "asc" |
+| taskDefaultFilterList | list | optional | Prop to pass default filterList available for any user and 
```

Comment:
Mention is it string or array . Sample prop data

Reply:
Updated with example and correct datatype @abhilash-aot